sentiment_with_wordcloud.Rmd

```diff
@@ -0,0 +1,86 @@
+# Visualizing Movie Reviews in Word Cloud
+
+Samrat Halder and Hariz Johnson
+
+```{r , include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+## IMDB Reviews
+We first scrape all the movie reviews from IMDB 
+```{r}
+library(rvest)
+library(xml2)
+library(plyr)
+result <- c()
+ID <- 4154796
+movie_IMDb <- read_html(paste0("https://www.imdb.com/title/tt",ID,"/reviews?ref_=tt_urv"))
+reviews <- movie_IMDb %>% html_nodes("#pagecontent") %>% html_nodes("div.content") %>% html_text()
+#perfrom data cleaning on user reviews
+reviews <- gsub("\r?\n|\r", " ", reviews) 
+reviews <- tolower(gsub("[^[:alnum:] ]", " ", reviews))
+write.csv(reviews, "reviews.csv")
+head(reviews)
+```
+## Cleaning the data!
+In this part first we clean the data using standard text mining techniques eg. removing redundant characters, stop words, stemming etc
+
+```{r}
+library(tm)
+library(plyr)
+library(stringr)
+library(wordcloud)
+library(RColorBrewer)
+
+removeURL <- function(x) {
+  gsub("http[[:alnum:]]*", "", x)
+}
+removelongWORDS <- function(x) {
+  gsub("\\b[[:alpha:]]{15,}\\b", "", x, perl=FALSE)
+}
+removeCharacters <-function (x, characters)  {
+  gsub(sprintf("(*UCP)(%s)", paste(characters, collapse = "|")), "", x, perl = FALSE)
+}
+reviews <- read.csv("reviews.csv", row.names = NULL)
```

Comment:
You should upload the dataset as well.

Reply:
Added the files to resources. Please check and let me know if now it is fine.